packages/config/src/android/Manifest.ts

```diff
@@ -209,3 +209,18 @@ export function removeMetaDataItemFromMainApplication(mainApplication: any, item
   }
   return mainApplication;
 }
+
+export function getMainApplicationMetaDataValue(
+  androidManifest: Document,
+  name: string
+): string | null {
+  const mainApplication = getMainApplication(androidManifest);
+
+  if (mainApplication?.hasOwnProperty('meta-data')) {
```

Comment:
```suggestion
  if (mainApplication?.['meta-data']) {
```

Reply:
rest of the code here is using hasOwnProperty so I'm assuming that it's intentional

Reply:
🤷 but I think it behaves the same and it's easier to read.